rfcs/0001-flux-operator/README.md

```diff
@@ -178,6 +179,43 @@ Events example:
   Warning  UpgradePending           25s   flux-operator    Upgrade to latest version 2.4.0 blocked by semver range 2.3.x
 ```
 
+### Sync configuration
+
+The `.spec.sync` field is optional and specifies the Flux sync configuration.
+When set, a Flux source and a Flux Kustomization are generated to sync
+the cluster state with the source repository.
+
+Sync fields:
+
+- `kind`: The source kind, supported values are `GitRepository`, `OCIRepository` and `Bucket`.
+- `url`: The URL of the source repository, can be a Git repository HTTP/S or SSH address, an OCI repository address or a Bucket endpoint.
+- `ref`: The source reference, can be a Git ref name e.g. `refs/heads/main`, an OCI tag e.g. `latest` or a Bucket name.
+- `path`: The path to the source directory containing the kustomize overlay or plain Kubernetes manifests to sync from.
```

Comment:
I have written a question about it in the implementation PR, but what about `LayerSelector`? I would rather use that than path for an `oci Artifact`.

Reply:
`LayerSelector` could be added via a kustomize patch if anyone needs it, for now there is no practical use to it. The artifacts in this case are create with `flux push artifact` for the fleet Git repo content, so path is a must for any of this to work.